zerver/webhooks/github/doc.md

```diff
@@ -1,25 +1,42 @@
+# Zulip GitHub integration
+
 Get GitHub notifications in Zulip!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
-
-    You can refer to GitHub's documentation for [webhook events](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads).
+1. {!generate-webhook-url-basic.md!}
 
-    {!git-webhook-url-with-branches.md!}
+1. On your repository's web page, go to **Settings**. Select **Webhooks**,
+   and click **Add webhook**. GitHub may prompt you for your password.
 
-1. Go to your repository on GitHub and click on the **Settings** tab.
-   Select **Webhooks**. Click on **Add webhook**. GitHub may prompt
-   you for your password.
+1. Set **Payload URL** to the URL generated above. Set **Content type**
+   to `application/json`. Select the events you'd like to be notified
+   about, and click **Add Webhook**.
 
-1. Set **Payload URL** to the URL constructed above. Set **Content type**
-   to `application/json`. Select the events you would like to receive
-   notifications for, and click **Add Webhook**.
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/github/001.png)
 
-See also the [GitHub Actions integration](/integrations/doc/github-actions).
+{% if all_event_types is defined %}
+
+{!event-filtering-additional-feature.md!}
+
+{% endif %}
+
+### Configuration options
+
+{!git-branches-additional-feature.md!}
+
+### Related documentation
+
+- [GitHub's documentation for webhook events](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads)
```

Comment:
Could be shorted a bit to "GitHub's webhook events documentation" and I'd make the link a reference link since it's so long, e.g. `[github-webhook-events]: URL` at the end of the file.